src/Components/ResetPassword/RestPassword.tsx

```diff
@@ -1,7 +1,104 @@
-import React from 'react'
+import { Typography, TextField, Button, Grid } from '@mui/material';
+import styles from '../SignUp/SignUp.module.scss';
+import signUpImage from "../../assets/Rectangle 8.png"
+import { useForm } from 'react-hook-form';
+import { Link, useNavigate } from 'react-router-dom';
+import axios from 'axios';
 
 export default function RestPassword() {
+
+  const navigate = useNavigate()
+
+  const {
+    register,
+    formState: { errors },
+    handleSubmit
+  } = useForm({ mode: "all" })
+
+  const onSubmit = (data) => {
+    console.log(data);
+    
+    axios.post("http://154.41.228.234:3000/api/v0/admin/users/reset-password", data
+    ).then(response => {
+      console.log(response.data);
+    })
+      .catch(error => {
+        console.error(error.data.data.message);
+      });
+      navigate("/signin")
```

Comment:
This need to be moved to the `then`  block to ensure that the request is successful.